install_config/install/advanced_install.adoc

```diff
@@ -316,7 +321,84 @@ openshift_master_session_encryption_secrets=['DONT+USE+THIS+SECRET+b4NV+pmZNSO']
 ----
 ====
 
+[[advanced-install-custom-certificates]]
+*Configuring Custom Certificates*
+
+link:../../install_config/certificate_customization.html[Custom serving
+certificates] for the public host names of the OpenShift API and
+link:../../architecture/infrastructure_components/web_console.html[web console]
+can be deployed during an advanced installation and are configurable in the
+inventory file.
+
+[NOTE]
+====
+Custom certificates should only be configured for the host name associated with
+the `*publicMasterURL*` which can be set using
+`*openshift_master_cluster_public_hostname*`. Using a custom serving certificate
+for the host name associated with the `*masterURL*`
+(*`openshift_master_cluster_hostname`*) will result in TLS errors as
+infrastructure components will attempt to contact the master API using the
+internal `*masterURL*` host.
+====
+
+Certificate and key file paths can be configured using the
+`*openshift_master_named_certificates*` cluster variable:
+
+====
+----
+openshift_master_named_certificates=[{"certfile": "/path/to/custom1.crt", "keyfile": "/path/to/custom1.key"}]
+----
+====
+
+File paths must be local to the system where Ansible will be run. Certificates
+are copied to master hosts and are deployed within the
+*_/etc/origin/master/named_certificates/_* directory.
+
+Ansible detects a certificate's `Common Name` and `Subject Alternative Names`.
+Detected names can be overridden by providing the `*"names"*` key when setting
+`*openshift_master_named_certificates*`:
+
+====
+----
+openshift_master_named_certificates=[{"certfile": "/path/to/custom1.crt", "keyfile": "/path/to/custom1.key", "names": ["public-master-host.com"]}]
+----
+====
 
+Certificates configured using `*openshift_master_named_certificates*` are cached
+on masters, meaning that each additional Ansible run with a different set of
+certificates results in all previously deployed certificates remaining in place
+on master hosts and within the master configuration file.
+
+If you would like `*openshift_master_named_certificates*` to be overwritten with
+the provided value (or no value), specify the
+`*openshift_master_overwrite_named_certificates*` cluster variable:
+
+====
+----
+openshift_master_overwrite_named_certificates: true
+----
+====
```

Comment:
Yeah, this new section about defining custom certs is within the larger http://file.rdu.redhat.com/~adellape/010816/customcert_edits/install_config/install/advanced_install.html#configuring-ansible section, which is all about setting variables in an inventory file (e.g., /etc/ansible/hosts).